### DISCUSSION TITLE
What happens if Github doesn't review Student Developer Pack application within deadline?

### DISCUSSION THREAD

**Discussion by mtatko** [community/community#22802] [New to GitHub]
<p>Hey! I have a little question.<br>
I’ve applied for Github Student Developer Pack back on the 1st of September. I’ve received no further information from them. My deadline is October 6th. My friend was accepted within 15 days of applying and I’m still pending. What can I do to get my application reviewed?<br>
Thanks in advance for any answers.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Because it’s a busy time of year, applications are taking up to 6 weeks to process. Different applications can involve different review processes depending on the details.</p>
<p>Hopefully you’ll hear soon.</p>

**Comment by mtatko** [comment]
<p>I understand, I can wait. But I would like one thing - why does Github Support ignore my emails? When I tried to ask the same question I got no response other than the autoresponder. I think ignoring emails is not nice.</p>

**Comment by mtatko** [comment]
<p>My application was approved, topic can be closed.</p>
